api-schema/src/main/java/org/stellar/anchor/api/platform/GetTransactionResponse.java

```diff
@@ -12,4 +12,6 @@
  */
 @SuperBuilder
 @NoArgsConstructor
-public class GetTransactionResponse extends PlatformTransactionData {}
+public class GetTransactionResponse extends PlatformTransactionData {
+  String fundingMethod;
```

Comment:
Is there any reason not to add this field directly to the `PlatformTransactionData` class? Also, do we plan to remove the `type` field from that class as we no longer set it?

Reply:
Why do we want to add an extra field to `PlatformTransactionData ` ? Since they are referencing the same thing I though the best way is to rename it.

But since the field has a bad name `type` I'm not gonna touch it here. Will try in a separate PR